recipes/dlib/lapack_sgetrf.patch

```diff
@@ -0,0 +1,20 @@
+--- dlib/cmake_find_blas.txt
++++ dlib/cmake_find_blas.txt
+@@ -140,15 +140,8 @@
+          set(blas_found 1)
+          message(STATUS "Found OpenBLAS library")
+          set(CMAKE_REQUIRED_LIBRARIES ${blas_libraries})
+-         # If you compiled OpenBLAS with LAPACK in it then it should have the
```

Comment:
Does it actually work with that copy of openblas that is missing symbols?
I've added checks like that specifically because some systems include
broken copies of various libraries which eventually cause linker errors due
to missing symbols.  Does that openblas actually have lapack in it?